prime-router/src/main/kotlin/azure/observability/bundleDigest/FhirPathBundleDigestLabResultExtractorStrategy.kt

```diff
@@ -10,24 +12,26 @@ class FhirPathBundleDigestLabResultExtractorStrategy(private val context: Custom
 
     private val eventCodePath = "Bundle.entry.resource.ofType(MessageHeader).event.display"
     private val patientStatePath = "Bundle.entry.resource.ofType(Patient).address.state"
-    private val performerStatePath =
-        "Bundle.entry.resource.ofType(ServiceRequest)[0].performer.resolve().address.state"
-    private val orderingFacilityStatePath =
-        "Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state"
+    private val performerPath =
+        "Bundle.entry.resource.ofType(ServiceRequest)[0].performer.resolve()"
```

Comment:
Doing manual testing right now but just wanted to report some findings. Apparently we do not map ServiceRequest.performer (and it is not in the mapping inventory, I think. ServiceRequest can be a little tricky with ORU messages). So performer state was never being returned and this won't return anything either.

Reply:
Tracked this down and it appears this is a field that SimpleReport populates. ServiceRequest.performer is called out in the original message monitoring docs for this, but the issue is that no other senders will have data for that field. We can follow up next week when Victor is back to see if this needs to be expanded to handle logging for more than the SimpleReport use case.